### PR TITLE
group 17 shiny_tutorial

### PR DESCRIPTION
**Pull request by ZujunPeng**

### CODE REVIEW

File: shiny_tutorial.Rmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+```{r  include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
```
Comment (eric0871) [inline shiny_tutorial.Rmd]: This part should not exist before the header.

**Review by eric0871 — CHANGES_REQUESTED** [review]
This file needs some changes before merging. Please see comment for detail.

File: shiny_tutorial.Rmd (hunk 1 above)
Comment (ZujunPeng) [reply shiny_tutorial.Rmd]: Problem is fixed now. Do I need to submit another pull request? Thank you!

**Comment by ybliu9** [comment]
@eric0871 Problem fixed, should be ready for merge:)

File: shiny_tutorial.Rmd (hunk 1 above)
Comment (tracy3057) [reply shiny_tutorial.Rmd]: No, you don't need to. Thank you!

**Comment by tracy3057** [comment]
Thank you for your contribution! 🎉

**Pull request merged**